src/engine/Actor.ts

```diff
@@ -1229,6 +1227,11 @@ module ex {
           }
        }
 
+       // Capture old values before integration step updates them
+       this.oldVel.setTo(this.vel.x, this.vel.y);
+       this.oldPos.setTo(this.pos.x, this.pos.y);
+
+       // Run Euler integration
        this.integrate(delta);
```

Comment:
@eonarheim is this a problem? Scene will call this once, then will call it again later multiple times. I don't know enough about how the physics work to know if that's bad.